recipe/meta.yaml

```
@@ -39,11 +39,12 @@ requirements:
     - setuptools
     - cython >=0.29,<0.30
     - spdlog >=1.8.5,<2
+    - cuda-python >=11.5,<12.0
```

Would suggest picking a version here like `11.5`

```suggestion
    - cuda-python 11.5
```

This way we build against a fixed lower bound version during the build, which is then used as a minimum in `run` with `pin_compatible`

---

Thanks, John, good point. But let's pin at a newer version of CUDA Python (which can still work with any `cudatoolkit=11.x`):
```suggestion
    - cuda-python 11.6
```